### PR TITLE
Updated import docs

### PR DESCRIPTION
**Pull request by shawntanzk**
See https://github.com/INCATools/ontology-development-kit/issues/514#issuecomment-1014272986

Sorry about that @bvarner-ebi & @dosumis - my bad for misunderstanding the import system. Fixed the docs accordingly here.

### CODE REVIEW

File: template/_dynamic_files.jinja2 [hunk 1]
```diff
@@ -1083,15 +1083,11 @@ To check if your ontology uses this method, check src/ontology/{{ project.id }}-
...
 
-Start by declaring the term to be imported in Protege:
+To check if your ontology uses this method, check src/ontology/{{ project.id }}-odk.yaml to see if `use_base_merging: TRUE` is declared under `import_group`
 
-1. Open your ontology (edit file) in Protege (5.5+).
-1. Select 'owl:Thing'
-1. Add a new class as usual.
-1. Paste the _full iri_ in the 'Name:' field, for example, http://purl.obolibrary.org/obo/CHEBI_50906.
-1. Click 'OK'
+If your ontology uses Base Module approach, please use the following steps:
```
Comment (gouttegd) [inline template/_dynamic_files.jinja2:1088]: Lines 1086–1088 are duplicate of lines 1082–1084.

**Review by gouttegd — CHANGES_REQUESTED** [review]
Looks good to me except for the duplicated lines around 1082–1084.

File: template/_dynamic_files.jinja2 (hunk 1 above)
Comment (shawntanzk) [reply template/_dynamic_files.jinja2:1088]: whoops, my bad, i accidentally commited on a dirty branch, and was moving things around and missed that. Thanks for spotting :)

**Comment by matentzn** [comment]
I am not sure why the instruction for the Protege based approach are removed entirely.. How should people work without access to docker/odk? Are these instructions going somewhere else?

**Comment by shawntanzk** [comment]
> I am not sure why the instruction for the Protege based approach are removed entirely.. How should people work without access to docker/odk? Are these instructions going somewhere else?

ok looking at the diff for this, something looks odd -let me go relook at the whole thing again and redo this properly. (there was some removal cause there was duplicated things, but it seems that it ended up fully removing it. Sorry about that will fix

**Comment by shawntanzk** [comment]
@matentzn ok looked at the whole docs, the protege-based declaration is still up top, however I removed it from the base file approach after recommendation that that is not great a way. With base file approach, ODK is kinda needed anyway no? (eg if someone is adding using the protege-based declaration, they have the instructions up top, but if someone is working with base-file, they would not need to do with protege-based declaration.) 
Happy to add back the protege-based declaration to the base-file section if you think its useful there too

**Comment by matentzn** [comment]
Regardless of base or anything else, adding a term can be done using Protege if necessary, even if that is discouraged. In order to actual _run_ the import (import the term, not just declare it (step1)), you need ODK in both cases!

**Comment by shawntanzk** [comment]
> Regardless of base or anything else, adding a term can be done using Protege if necessary

This step is already present in ln949 - should I copy and paste it somewhere around 1084?

**Comment by matentzn** [comment]
No, if its clear that it applies to all approaches, then no need for duplication

**Comment by shawntanzk** [comment]
I'll make sure it is :)

**Comment by shawntanzk** [comment]
ready to go - will merge after a thumbs up

**Review by gouttegd — APPROVED** [review]

**Pull request merged**